src/components/dashboard/Claim.js

```diff
@@ -35,13 +35,13 @@ class Claim extends Component<ClaimProps, ClaimState> {
 
   interval = null
 
-  goodWalletWrapped = wrapper(goodWallet, this.props.store)
+  goodWalletWrapped = wrapper(gdWallet, this.props.store)
 
   async componentDidMount() {
     //if we returned from facerecoginition then the isValid param would be set
     //this happens only on first claim
     const isValid = this.props.screenProps.screenState && this.props.screenProps.screenState.isValid
-    if (isValid && (await goodWallet.isCitizen())) {
+    if (isValid && gdWallet && (await gdWallet.isCitizen())) {
```

Comment:
we solved the naming issue in a better way. make sure this doesnt overwrite master